aces/Loris
# [Imaging Uploader] Fix URL to MRI violation module

## Brief summary of changes

This PR fixes the URL to the MRI violation module from the Imaging Uploader so that the filter for PatientName works with the new Reactified MRI violation module.

#### Testing instructions

1. Checkout this branch
2. Click on a link with violations and check that the results int he MRI violation module is filter for the PatientName

#### Link(s) to related issue(s)

* Resolves #8665
